src/helm/helmfile.yaml

```diff
@@ -45,9 +50,50 @@ releases:
           enabled: true
           autoGenerated: true
 
+  - name: keycloak
+    installed: {{ or (eq .Environment.Name "dev") (eq .Environment.Name "dev-keycloak") | toYaml }}
+    missingFileHandler: Warn
+    namespace: {{ .Namespace }}
+    chart: bitnami/keycloak
+    version: 17.3.6
+    values:
+      - postgresql:
+          auth:
+            username: keycloak
+            password: keycloak
+            databaese: keycloak
+      - extraEnvVars:
+          - name: KEYCLOAK_EXTRA_ARGS
+            value: "--import-realm"
+          - name: KC_HOSTNAME_URL
+            value: https://keycloak.127.0.0.1.nip.io
+      - extraVolumes:
+          - name: import
+            configMap:
+              name: meet-keycloak
+      - extraVolumeMounts:
+          - name: import
+            mountPath: /opt/bitnami/keycloak/data/import/
+      - auth:
+          adminUser: su
+          adminPassword: su
+      - proxy: edge
+      - ingress:
+          enabled: true
+          hostname: keycloak.127.0.0.1.nip.io
+      - extraDeploy:
+        - apiVersion: v1
+          kind: ConfigMap
+          metadata:
+            name: meet-keycloak
+          data:
+            meet.json: |
+{{ readFile "../../docker/auth/realm.json" | replace "http://localhost:3200" "https://meet.127.0.0.1.nip.io" | indent 14 }}
+
   - name: minio
-    installed: {{ eq .Environment.Name "dev" | toYaml }}
+    installed: {{ or (eq .Environment.Name "dev") (eq .Environment.Name "dev-keycloak") | toYaml }}
```

Comment:
would smth as: 
```
{{ contains .Environment.Name "dev" | toYaml }}
```
I've not tested it

Reply:
could we even do a yaml function, with an explicit name, `isDevEnvironment` ? wdyt ?